discovery-provider/src/api/v1/reactions.py

```diff
@@ -0,0 +1,50 @@
+from flask_restx import Namespace, Resource, fields, marshal_with, reqparse
+from src.api.v1.helpers import (
+    DescriptiveArgument,
+    extend_reaction,
+    make_response,
+    success_response,
+)
+from src.api.v1.models.reactions import reaction
+from src.queries.reactions import get_reactions
+from src.utils.db_session import get_db_read_replica
+from src.utils.redis_cache import cache
+from src.utils.redis_metrics import record_metrics
+
+ns = Namespace("reactions", description="Reaction related operations")
+
+get_reactions_parser = reqparse.RequestParser(argument_class=DescriptiveArgument)
+get_reactions_parser.add_argument(
+    "type", required=True, description="The type of reactions for which to query."
+)
+get_reactions_parser.add_argument(
+    "tx_id",
```

Comment:
Guess I should update my PR to use `tx_id` to be consistent? Right now I call mine `transaction_signatures`

Reply:
chatted on slack - going with `tx_signatures`